src/WebJobs.Script.Abstractions/Description/Binding/BindingMetadata.cs

```diff
@@ -81,6 +87,10 @@ public static BindingMetadata Create(JObject raw)
             string bindingDirectionValue = (string)raw["direction"];
             string connection = (string)raw["connection"];
             string bindingType = (string)raw["type"];
+            var properties = raw["properties"] != null
+                                ? raw["properties"].ToObject<IDictionary<string, object>>()
+                                : new Dictionary<string, object>();
```

Comment:
I get an error when I try this, doesn't seem happy with the expression 🤔

```
Type of conditional expression cannot be determined because there is no implicit conversion between 'System.Collections.Generic.Dictionary<string, object>' and 'System.Collections.Immutable.ImmutableDictionary<string, object>' 
```